integrations/PinterestTag/browser.js

```diff
@@ -193,14 +191,22 @@ export default class PinterestTag {
         };
       }
     }
+    const destinationEvent = eventMapping.find((p) =>
+      p.src.includes(event.toLowerCase())
+    );
+    if (!destinationEvent) {
+      return {
+        dest: "custom",
+      };
```

Comment:
Does Pinterest support custom?

Reply:
I can see custom call in their dashboard.
<img width="1085" alt="Screenshot 2022-09-01 at 11 58 23 AM" src="https://user-images.githubusercontent.com/60211312/187846665-ef24e769-ed74-4964-b070-fbce238ef092.png">

Reply:
ok